src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/EnumBuilder.cs

```diff
@@ -46,7 +46,7 @@ public FieldBuilder DefineLiteral(string literalName, object? literalValue)
 
```

Comment:
Also change `public TypeInfo? CreateTypeInfo()` to `public TypeInfo CreateTypeInfo()`

Reply:
This has been updated.